src/test/java/hudson/plugins/git/GitSCMTest.java

```diff
@@ -1218,6 +1218,20 @@ public void testEnvironmentVariableExpansion() throws Exception {
         assertTrue(r.hasChanges());
 
         build(project, Result.SUCCESS, "b.txt");
+
+        // Test branch expansion with environment contributors
+        FreeStyleProject project2 = createFreeStyleProject();
```

Comment:
Why split it?  They're both testing environment expansion with EnvironmentContributors, we just forgot to check it in the branch as well as the repo.